.scalafmt.conf

```diff
@@ -1,5 +1,5 @@
 version = 2.4.1
 align = more    // For pretty alignment.
 maxColumn = 100 // For my wide 30" display.
-rewrite.rules = [SortImports]
```

Comment:
Quick question about this -- the scalafix rule does block sorting, and scalafmt did sorting within lines i thought like 

```diff
- import doobie.{Transactor, ConnectionIO}
+ import doobie.{ConnectionIO, Transactor}
```

is that not right?

Reply:
Oh, I didn't realize that was doing anything -- I'll add it back.

Reply:
I added it back but it didn't change anything - `SortImports` sorts by:

> symbols, lower-case, upper-case.

So in this case they're equivalent